tf-proj/main.tf

```diff
@@ -2,32 +2,10 @@ terraform {
   required_providers {
     buildkite = {
       source  = "buildkite/buildkite"
-      version = "0.12.0"
+      version = "0.11.1"
     }
   }
 }
 
 provider "buildkite" {
-}
```

Comment:
I think leave this stuff in for future testing (we can just play with this file locally and not commit it when testing)